test/Serilog.Tests/Core/LoggerTests.cs

```diff
@@ -317,6 +317,175 @@ public void NullMessageTemplateParametersDoNotBreakBinding()
         // ReSharper restore StructuredMessageTemplateProblem
     }
 
+    // https://github.com/serilog/serilog/issues/2019
+    [Fact]
+    public void Two_Dimensional_Array_Should_Be_Logger_As_Sequence()
```

Comment:
Nit: `_Should_Be_Logger` => `_Should_Be_Logged` typo in this and several other of the test names.

Naming: tests in this project use PascalCase names, so `TwoDimensionalArrayShouldBeLoggedAsSequence` would be the matching casing here.

Reply:
Fixed Naming and Linting

Reply:
Thanks! Merged :+1: